webpack.config.js

```diff
@@ -0,0 +1,233 @@
+      { parser: { requireEnsure: false } },
+      // Process JS with Babel.
+      {
+        test: /\.js$/,
+        include: sourceFolders,
+        use: [
+          {
+            loader: require.resolve('babel-loader'),
+            options: {
+              babelrc: false,
+              compact: false,
+              presets: [
+                require.resolve('@commercetools-frontend/babel-preset-mc-app'),
+              ],
+              // This is a feature of `babel-loader` for webpack (not Babel itself).
+              // It enables caching results in ./node_modules/.cache/babel-loader/
+              // directory for faster rebuilds.
+              cacheDirectory: true,
+              highlightCode: true,
+            },
+          },
+        ],
+      },
+      // For svg icons, we want to get them transformed into React components
+      // when we import them.
+      {
+        test: /\.react\.svg$/,
+        include: sourceFolders,
+        use: [
+          {
+            loader: require.resolve('babel-loader'),
+            options: {
+              babelrc: false,
+              presets: [
+                require.resolve('@commercetools-frontend/babel-preset-mc-app'),
+              ],
+              // This is a feature of `babel-loader` for webpack (not Babel itself).
+              // It enables caching results in ./node_modules/.cache/babel-loader/
+              // directory for faster rebuilds.
+              cacheDirectory: true,
+              highlightCode: true,
+            },
+          },
+          {
+            loader: require.resolve('@svgr/webpack'),
+            options: {
+              // NOTE: disable this and manually add `removeViewBox: false` in the SVGO plugins list
+              // See related PR: https://github.com/smooth-code/svgr/pull/137
+              icon: false,
+              svgoConfig: {
+                plugins: [
+                  { removeViewBox: false },
+                  // Keeps ID's of svgs so they can be targeted with CSS
+                  { cleanupIDs: false },
+                ],
+              },
+            },
+          },
+        ],
+      },
+      // For normal svg files (not icons) we should load the file normally
+      // and simply use it as a `<img src/>`.
+      {
+        test: function testForNormalSvgFiles(fileName) {
+          return (
+            // Use this only for plain SVG.
+            // For SVG as React components, see loader above.
+            fileName.endsWith('.svg') && !fileName.endsWith('.react.svg')
+          );
+        },
+        use: [
+          {
+            loader: require.resolve('svg-url-loader'),
+            options: { noquotes: true },
+          },
+        ],
+      },
+      // "url" loader works like "file" loader except that it embeds assets
+      // smaller than specified limit in bytes as data URLs to avoid requests.
+      // A missing `test` is equivalent to a match.
+      {
+        test: /\.png$/,
```

Comment:
I would remove this IMO we shouldn't be bundling images with UI-kit.

Reply:
Good point